source/upgrading/upgrade/upgrade-4.13.rst

```diff
@@ -0,0 +1,360 @@
+
+   .. parsed-literal::
+
+      $ mysql -u cloud -p -e 'update cloud.storage_pool set path="/var/lib/libvirt/images" where path="/var/lib/libvirt/images/"';
+
+
+.. _ubuntu412:
+.. _apt-repo412:
+
+Management Server
+-----------------
+
+Ubuntu
+######
+
+If you are using Ubuntu, follow this procedure to upgrade your packages. If
+not, skip to step :ref:`rhel411`.
+
+.. note::
+   **Community Packages:** This section assumes you're using the community
+   supplied packages for CloudStack. If you've created your own packages and
+   APT repository, substitute your own URL for the ones used in these examples.
+
+The first order of business will be to change the sources list for
+each system with CloudStack packages. This means all management
+servers, and any hosts that have the KVM agent. (No changes should
+be necessary for hosts that are running VMware or Xen.)
+
+
+
+Start by opening ``/etc/apt/sources.list.d/cloudstack.list`` on
+any systems that have CloudStack packages installed.
+
+This file should have one line, which contains:
+
+.. parsed-literal::
+
+   deb http://download.cloudstack.org/ubuntu precise 4.12
+
+We'll change it to point to the new package repository:
+
+.. parsed-literal::
+
+   deb http://download.cloudstack.org/ubuntu precise |version|
+
+Setup the public key for the above repository:
+
+.. parsed-literal::
+
+   wget -qO - http://download.cloudstack.org/release.asc | sudo apt-key add -
+
+If you're using your own package repository, change this line to
+read as appropriate for your |version| repository.
+
+#. Now update your apt package list:
+
+   .. parsed-literal::
+
+      $ sudo apt-get update
+
+#. Now that you have the repository configured, it's time to upgrade
+   the ``cloudstack-management`` package.
+
+   .. parsed-literal::
+
+      $ sudo apt-get upgrade cloudstack-management
+
+#. If you use CloudStack usage server
+
+   .. parsed-literal::
+
+      $ sudo apt-get upgrade cloudstack-usage
+
+
+.. _rhel412:
+.. _rpm-repo412:
+
+CentOS/RHEL
+##############
+
+If you are using CentOS or RHEL, follow this procedure to upgrade your
+packages. If not, skip to hypervisors section :ref:`upg_hyp_412`.
+
+.. note::
+   **Community Packages:** This section assumes you're using the community
+   supplied packages for CloudStack. If you've created your own packages and
+   yum repository, substitute your own URL for the ones used in these examples.
+
+The first order of business will be to change the yum repository
+for each system with CloudStack packages. This means all
+management servers, and any hosts that have the KVM agent.
+
+(No changes should be necessary for hosts that are running VMware
+or Xen.)
+
+Start by opening ``/etc/yum.repos.d/cloudstack.repo`` on any
+systems that have CloudStack packages installed.
+
+This file should have content similar to the following:
+
+.. parsed-literal::
+
+   [apache-cloudstack]
+   name=Apache CloudStack
+   baseurl=http://download.cloudstack.org/centos/6/4.12/
```

Comment:
http://download.cloudstack.org/centos/**7**/4.13/

We need to add an explicit warning that 4.14 will no more run on CentOS6

Reply:
It's again saying to change this line to the new repo.

Reply:
fixed